spec/jobs/destroy_message_job_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DestroyMessageJob, type: :job do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Check this gem for writing test about ActionCable 

https://github.com/palkan/action-cable-testing (after rails 6 this gem is merged into Rails repo)